etc/dev-bin/create-version-file

```diff
@@ -8,7 +8,7 @@ cd "${THIS_DIR}/../.."
 if [[ -d .git || -f .git ]]; then
   echo "(a cylc VERSION file is not needed in a cylc git repo)" >&2
 else    
-  VN=${PWD#*cylc-}
+  VN=${PWD#*cylc-flow-}
```

Comment:
For those who are stuck in the old way, (sorry), perhaps this is friendlier?

```suggestion
  VN=${PWD#*cylc-}
  VN=${VN#*flow-}
```

Reply:
Fair enough.